### ISSUE TITLE
Can I train roBERTa on my dataset of sentences in order to find out the most similar sentences compared to a custom sentence? 

### ISSUE THREAD

**Issue by chamecall** [pytorch/fairseq#1255]
Can I train roBERTa on my dataset of sentences in order to find out the most similar sentences compared to a custom sentence? 
If so then how can I approach that?
10x in advance.

**Comment by ngoyal2707** [comment]
@chamecall RoBERTa is basically a sentence encoder that you can use to get good vector representation for any sentence.

Please look at `STS-B` task from `GLUE` and released roberta glue section. That task is about predicitng sentence similarity. So you can use the same loss and setup as that task for finetuning RoBERTa on your task.
